contribs/gmf/src/datasource/Manager.js

```diff
@@ -847,14 +843,11 @@ class Manager {
 
     const id = olUtilGetUid(dataSource.gmfLayer);
     const item = this.treeCtrlCache_[id];
-    googAsserts.assert(item);
-    const wmsLayer = googAsserts.assert(item.wmsLayer);
-    const wmsSource = googAsserts.assertInstanceof(
-      wmsLayer.getSource(),
-      olSourceImageWMS
-    );
+    console.assert(item);
+    const wmsLayer = console.assert(item.wmsLayer);
```

Comment:
same issue here. And everywhere we have `= console.assert`

Reply:
Thanks fixed, I hope all :-)